WBooks/Book Comments/BookCommentsViewController.swift

```diff
@@ -33,6 +33,9 @@ class BookCommentsViewController: UIViewController {
     }
     
     private func configureCommentsTableView() {
+        _viewModel.comments.producer.startWithValues { _ in
+            self._view.comments.reloadData()
```

Comment:
When you use `self` in closures you need to set ir as `weak` of `unowned` to avoid retain cycles:
`.startWithValues { [weak self] _ in self?._view.comments.reloadData() }